app/jobs/report_usage_job.rb

```diff
@@ -0,0 +1,7 @@
+class ReportUsageJob < ApplicationJob
```

Comment:
As it is now, it looks like this'll never get enqueued in production. Are we going to create something to randomly enqueue metrics for active `usage` plans?

Reply:
yeah this is just for demo-ing purposes so here are my thoughts on making it a background job that is not run on prod:
- since it's already a job, it will be easy to set it up to run via scheduler when we are ready for that
- if we added it to scheduler now, it wouldn't end up doing anything since we are not generating `usage` records in any non-manual way in sudo-sandwich
- getting all of this to work properly requires usage records with proper timestamps since usage records can only be uploaded for the previous hour...which reminds me that maybe we should create a rake task in here to generate that data automatically? Doing it by hand is not impossible but a little painful.